source/tools/amico.rst

```diff
@@ -0,0 +1,8 @@
+Amico / Commit
+==============
+
+To Do. Can you could help me?
```

Comment:
@arnaudbore do you have a description or ideas to what we could say about amico/commit ?